packages/core/src/fhir-to-cda/cda-templates/components/medications.ts

```diff
@@ -95,17 +92,19 @@ function createTableRowsFromMedicationStatement(
   };
   const code = buildCodeCeFromCoding(statement.medication?.code?.coding);
   const medicationName = statement.medication?.code?.text ?? code?._displayName;
-  const medicationCode = buildMedicationCode(code);
   return [
     {
       tr: {
         _ID: referenceId,
         ["td"]: [
           {
-            "#text": medicationName,
+            "#text": medicationName ?? NOT_SPECIFIED,
+          },
+          {
+            "#text": code?._code,
           },
           {
-            "#text": medicationCode, // TODO: Improve this to show the human readable system name alongside the system
+            "#text": code?._codeSystem ?? code?._codeSystemName,
```

Comment:
Well, my thinking for putting this here is that HTML tables of CDAs are for human eyes to interpret stuff. So it might be easier for ppl to see the human readable name rather than the OID. But because Epic seems to be working with OIDs, the name is just a backup

Reply:
Can't this element have multiple `<text></text>`? If so, maybe we could try to include both, as it seems it would allow Epic to parse it, as well as "follow the standard" in the way it helps to read that if they get directly displayed to people?

Reply:
Are you sure that's what the html tables of CDA are for? It sounded from the email like they parsed those? 

If it is the case that the html is just for the human readable version, then maybe this change doesn't even address the issue epic was bringing up since there's was an issue in parsing code and code system?

Reply:
It is possible to have more than one coding there, but for the purposes of the table, it makes sense to keep just one. I don't think this is an issue.

Reply:
And yea, it does sound like they're parsing those. I guess I'll remove the codeSystemName